src/classes/project_data.py

```diff
@@ -256,7 +256,7 @@ def new(self):
             try:
                 self._data = self.read_from_file(info.USER_DEFAULT_PROJECT)
             except (FileNotFoundError, PermissionError) as ex:
-                log.warning("Unable to load user project defaults from {}: {}".format(info.USER_DEFAULT_PROJECT, ex))
+                log.debug("Unable to load user project defaults from %s: %s", info.USER_DEFAULT_PROJECT, ex)
```

Comment:
**Definitely** `log.warning`: This is a user-supplied file, **and** we've already checked that it exists. If we're now having trouble reading it, the issue is on their end (wrong file permissions, something like that). They need to know about it. However, I'd again go with `exc_info=1` for the exception details.